lib/carrierwave/processing/rmagick.rb

```diff
@@ -351,5 +373,9 @@ def destroy_image(image)
       image.destroy! if image.respond_to?(:destroy!)
     end
 
+    def rmagick_image
+      ::Magick::Image.ping(current_path).first
```

Comment:
I used `ping` instead of `read` as we are interested in the image metadata, but don't care about the pixel data.
Ref http://www.rubydoc.info/gems/rmagick/2.15.4/Magick/Image#ping-class_method